src/static/helpers/environment.test.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import nodeFetch from 'node-fetch';
+import {isNode, hasFetchAvailable, fetch} from './environment';
+
+/* Just testing the node environment, testing the browser environment is too complex within the test cases. */
+
+describe('Fetch', () => {
+  test('Runs node-fetch if node', () => {
+    expect(isNode).toBe(true);
+    expect(hasFetchAvailable).toBe(false);
```

Comment:
This'll break on node 18, but that's a future problem!

Reply:
I can mine as well remove that line, no need for future headaches.